packages/circuits/circom/utils/EdDSAPoseidonVerifier.circom

```diff
@@ -8,60 +8,69 @@ include "./bitify.circom";
 include "./escalarmulany.circom";
 include "./escalarmulfix.circom";
 // local imports
-include "./hashers.circom";
+include "./PoseidonHasher.circom";
 
 /**
  * Variant of the EdDSAPoseidonVerifier template from circomlib
  * https://github.com/iden3/circomlib/blob/master/circuits/eddsa.circom
  */
-template EdDSAPoseidonVerifier_patched() {
+template EdDSAPoseidonVerifier() {
+    // The prime subgroup order.
```

Comment:
the prime subgroup order is 2736030358979909402780800718157159386076813972158567259200215660948447373041